src/validate/html/index.ts

```diff
@@ -54,6 +54,12 @@ export default function validateHtml(validator: Validator, html: Node) {
 		if (node.else) {
 			visit(node.else);
 		}
+
+		if (node.type === 'AwaitBlock') {
```

Comment:
What happens if the await block is missing one of these sections? Would `visit` get called with undefined? It looks like that would throw an exception.

Reply:
That's never the case — see https://github.com/sveltejs/svelte/blob/master/src/parse/state/mustache.ts#L203-L229. If we changed how that worked in future, the tests would fail, so we'd be able to add guards at that point.